tasks/build_docker_image.yml

```diff
@@ -15,11 +15,15 @@
     path: "{{ lookup('env','PWD') }}"
     name: "{{ wimpy_docker_image_name }}:{{ wimpy_release_version | string }}"
     push: yes
+    force: "{{ wimpy_docker_image_force }}"
     buildargs: "{{ wimpy_docker_build_args }}"
 
 - name: "Build and push latest tag"
   docker_image:
     path: "{{ lookup('env','PWD') }}"
     name: "{{ wimpy_docker_image_name }}:latest"
     push: yes
+    force: "{{ wimpy_docker_image_force }}"
```

Comment:
Mmm.... I was just thinking. Should we always force this (instead of using variable)? I mean, in case you have `wimpy_docker_image_skip_latest_tag` set to False, then you most likely want to override the `latest` tag, right?